### PR TITLE
Exposed Notification Overlay position

### PR DESCRIPTION
**Pull request by nospoone**
I exposed a parameter to change the notification overlay position. 

I know next to nothing in CPP - I haphazardly got this in using [your guide](https://github.com/snowkit/hxcpp-guide/issues/1) (thanks!) and poking around.

### CODE REVIEW

**Comment by larsiusprime** [comment]
Just confirming, did you test this and did it work?

File: native/SteamWrap.cpp [hunk 1]
```diff
@@ -399,18 +399,36 @@ static bool CheckInit()
...
+		case 2:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomRight);
+			break;
+		case 3:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomLeft);
+			break;
+		default:
+			SteamUtils()->SetOverlayNotificationPosition(k_EPositionBottomRight);
+			break;
+		}
```
Comment (larsiusprime) [inline native/SteamWrap.cpp:427]: style note: if you could please indent the case statements and their bodies by one tab each so they start one tab in from the open & close brackets

File: steamwrap/api/Steam.hx [hunk 2]
```diff
@@ -10,6 +10,15 @@ private enum LeaderboardOp
...
+@:enum
+abstract SteamNotificationPosition(Int) to Int 
+{
+	var TopLeft = 0;
+	var TopRight = 1;
+	var BottomRight = 2;
+	var BottomLeft = 3;
+}
+
 typedef ControllerHandle = Int64;
```
Comment (larsiusprime) [inline steamwrap/api/Steam.hx:22]: Nice! Thanks for using an enum on the haxe side too, so we can keep the user from dealing with "magic numbers"

**Comment by nospoone** [comment]
I did! Worked fine in Holobunnies.

**Comment by larsiusprime** [comment]
Everything looks right, let's merge it.

File: native/SteamWrap.cpp (hunk 1 above)
Comment (nospoone) [inline native/SteamWrap.cpp:427]: whoops! fixed it.

**Pull request merged**